features/Hotels/Tc-21.feature

```diff
@@ -0,0 +1,10 @@
+#TC21
+Feature: Sign In
+
+    Scenario: Verify Verification message for invalid sign in credentials
+        Given I am on hotels website
+        When I click on the link of "Sign in" 
+        And Then I click on Sign in button
```

Comment:
Then is a Gherkin language key-word, it should not be part of a step itself like `And Then I click on Sign in button`